modules/virt-about-using-virtual-gpus.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-using-virtual-gpus_{context}"]
+= About using virtual GPUs with {VirtProductName}
+
+Some graphics processing unit (GPU) cards support the creation of virtual GPUs (vGPUs). {VirtProductName} can automatically create vGPUs and other _mediated devices_ if an administrator provides configuration details in the `HyperConverged` custom resource (CR). This automation is especially useful for large clusters.
+
+[NOTE]
+====
+Refer to your hardware vendor's documentation for functionality and support details.
+====
+
+Mediated device:: A physical device that is divided into one or more virtual devices. A vGPU is a type of mediated device (mdev); the performance of the physical GPU is divided among the virtual devices. You can assign mediated devices to one or more virtual machines (VMs), but the number of guests must be compatible with your GPU. Some GPUs do not support multiple guests.
+
+[id="configuration-overview_{context}"]
+== Configuration overview
+
+To configure mediated devices in the `HyperConverged` CR, an administrator must:
+
+* List the types of mediated devices that you want {VirtProductName} to create in the `spec.mediatedDevicesConfiguration` stanza.
+
+* Expose the mediated devices to the cluster by listing their details in the `spec.permittedHostDevices` stanza.
+
+Optionally, you can override the global configuration for a specific node or group of nodes by including the `spec.mediatedDevicesConfiguration.nodeMediatedDeviceTypes` stanza. This stanza must contain its own `mediatedDevicesTypes` list.
```

Comment:
We recently updated the [upstream user-guide](https://github.com/kubevirt/user-guide/blob/main/docs/operations/mediated_devices_configuration.md#mediated-devices-and-virtual-gpus), so referring the 2nd para from it, 

`It would also be possible to provide a more specific configuration that targets a specific node or a group of nodes directly. This can be achieved by using the nodeMediatedDeviceTypes, however, it must be used in a combination with mediatedDevicesTypes, to override the global configuration set in mediatedDevicesTypes section. Therefore, using nodeMediatedDeviceTypes without the mediatedDevicesTypes section is not supported.`

There are 2 things, 
a) This stanza must contain its own `mediatedDevicesTypes` list.
b) Therefore, using nodeMediatedDeviceTypes without the mediatedDevicesTypes section is not supported.

**Summary:** `nodeMediatedDeviceTypes` should be used along with `mediatedDevicesTypes` ( global config)  and also contain its own `mediatedDevicesTypes` list.